firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -379,6 +379,7 @@ extension BrowserViewController: WKNavigationDelegate {
         if let url = webView.url {
             guard !url.isReaderModeURL else { return }
             updateReaderModeState(for: tabManager.selectedTab, readerModeState: .unavailable)
+            guard !isToolbarRefactorEnabled else { return }
```

Comment:
What is the reason for this check? Not calling this when toolbar refactor is enabled could potentially cause issues with reader mode.